src/controllers/dynakube/oneagent/daemonset/arguments.go

```diff
@@ -36,7 +36,7 @@ func (dsInfo *builderInfo) appendNetworkZoneArg(args []string) []string {
 }
 
 func (dsInfo *builderInfo) appendProxyArg(args []string) []string {
-	if dsInfo.hasProxy() {
+	if !dsInfo.instance.FeatureOneAgentIgnoreProxy() && dsInfo.hasProxy() {
```

Comment:
can also be refactored into a Dynakube property. See the comment for the activegate proxy flag